test/ui_test.exs

```diff
@@ -0,0 +1,74 @@
+defmodule UITest do
+  use ExUnit.Case
+
+  import ExUnit.CaptureIO
+
+  test "introduces game" do
+    assert capture_io(fn ->
+      UI.introduce_ttt
+    end) == "THIS IS *** TIC-TAC-TOE ***\n"
+  end
+
+  test "asks who starts" do
+    assert capture_io([input: "x\n"], fn ->
+      UI.prompt_for_starter
+    end) == "Who starts? [enter X or O]\n"
+  end
+
+  test "registers and formats starter mark chosen by user" do
+    assert capture_io([input: "x\n"], fn ->
+      assert UI.prompt_for_starter == "X"
+    end)
+  end
+
+  test "prints board" do
+    assert capture_io(fn ->
+      board = %Board{size: 3, cells: Board.create_cells(3)}
+
+      assert UI.print_board(board)
+    end) == "1 | 2 | 3\n4 | 5 | 6\n7 | 8 | 9\n"
+
+  end
+
+  test "asks current player to choose a position" do
+    assert capture_io([input: "1\n"], fn ->
+      UI.prompt_for_position("X")
+    end) == "It's X's turn: choose a valid position on the board\n"
+  end
+
+  test "registers and formats position chosen by user" do
```

Comment:
This and the test above could be combined into one. The trick is to find the right words for it.
The arrangement and execution are the same, the only thing that changes is where you placed the assertions.